src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -103,9 +104,20 @@ private static ImmutableArray<string> GetRazorSemanticTokenTypes()
     }
 
     private static readonly string[] s_tokenModifiers = new string[] {
+        // TODO: These two should come from Roslyn once an available version is inserted
         // Razor
-        "None",
-        // C# Modifiers
         "static",
+        // C# Modifiers
+        "ReassignedVariable",
+        // Razor background
+        nameof(RazorTokenModifiers.RazorCode)
     };
+
+    [Flags]
+    public enum RazorTokenModifiers
+    {
+        // Must start after the last Roslyn modifier. So Roslyn has two modifiers right now, 1 and 2, so we start
+        // at 4 (noting this is a Flags enum)
+        RazorCode = 4
```

Comment:
I much prefer that syntax too, because my brain struggles with binary, but ironically it means I'm only good at copying the existing pattern once its already in use, because my brain struggles with binary :)